docs/development.md

```diff
@@ -142,6 +142,14 @@ Installs prometheus monitoring rules for `ssp-operator` metrics. The available
 metrics can be found in `docs/metrics.md`. The installed rules can be found in
 `internal/operands/metrics/resources.go`.
 
+#### `vm-delete-protection`
+
+Installs a [ValidatingAdmissionPolicy](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/)
+(VAP) and ValidatingAdmissionPolicyBinding which binds the VAP to all namespaces. The VAP prevents VirtualMachine (VM) objects
```

Comment:
`ValidatingAdmissionPolicyBinding (VAP Binding)` is missing a link to docs, maybe link it to [1]. 

[1] https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy

Reply:
Done.